assets/js/amp-admin-tables.js

```diff
@@ -0,0 +1,46 @@
+/* global jQuery, ampAdminTables */
+( function( $ ) {
+	'use strict';
+
+	var adminTables = {
+
+		load: function() {
+			$( document ).ready( $.proxy( function() {
+				this.sourcesHideShow();
+				this.allSourcesHideShow();
+				this.addViewErrorsByTypeLinkButton();
+				this.boldURLInPageTitle();
+			}, this ) );
+		},
+
+		sourcesHideShow: function() {
+			$( 'span.dashicons.toggle-sources' ).on( 'click', function() {
+				$( this ).next( '.sources-container' ).toggleClass( 'collapsed' );
+				$( this ).toggleClass( 'dashicons-arrow-down dashicons-arrow-up' );
+			} );
+		},
+
+		allSourcesHideShow: function() {
+			$( '.double-arrow' ).on( 'click', function() {
+				$( '.double-arrow' ).find( '.dashicons' ).toggleClass( 'dashicons-arrow-down dashicons-arrow-up' );
+				$( 'span.dashicons.toggle-sources' ).toggleClass( 'dashicons-arrow-down dashicons-arrow-up' );
+				$( '.sources-container' ).toggleClass( 'collapsed' );
+			} );
+		},
+
+		addViewErrorsByTypeLinkButton: function() {
+			$( '.wp-heading-inline' ).after( '<a href="' + ampAdminTables.errorsByTypeLink + '" class="page-title-action">View errors by <strong>Type</strong></a>' );
```

Comment:
The text here is not translatable. I suggest doing this: at the admin footer action whenever you are printing scripts, output right to the page the entire link tag with `esc_url()` and `esc_html_e()` and such. Give that link an ID and include a `hidden` attribute. Then in this `addViewErrorsByTypeLinkButton` function, just move the element in the DOM and remove the `hidden` attribute.